index.html

```diff
@@ -36,7 +36,217 @@ <h2 class="sd">
+                    <a href="#"><img src="./images/Indicator.png" alt="#"></a>
+                </div>
+                    <div class="container-card">
+                    <div class="placeholder">         
+                    </div>
+                    <div class="card-lists">
+                        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                        <div class="items">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>Css</li>
+                            <li>Java script</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <div class="button-cont">
+                        <input type="button" value="see project" class="potfolio-but">
+                    </div>
+
+                    </div>
+                </div>
+            <div class="container-card">
+            <div class="placeholder">
                 
+            </div>
+            <div class="card-lists">
+                <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                <div class="items">
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>Css</li>
+                    <li>Java script</li>
+                    <li>Html</li>
+                </ul>
+            </div>
+            <div class="button-cont">
+                <input type="button" value="see project" class="potfolio-but">          
+              </div>
+            </div>
+        </div>
+    <div class="container-card">
+    <div class="placeholder">
+        
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <input type="button" value="see project" class="potfolio-but">
+    </div>
+    </div>
+</div>
+<div class="container-card">
+<div class="placeholder">
+    
+</div>
+<div class="card-lists">
+    <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+    <div class="items">
+    <ul>
+        <li>Ruby on rails</li>
+        <li>Css</li>
+        <li>Java script</li>
+        <li>Html</li>
+    </ul>
+</div>
+<div class="button-cont">
+    <input type="button" value="see project" class="potfolio-but">
+</div>
+</div>
+</div>
+<div class="container-card">
+<div class="placeholder">
+    
+</div>
+<div class="card-lists">
+    <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+    <div class="items">
+    <ul>
+        <li>Ruby on rails</li>
+        <li>Css</li>
+        <li>Java script</li>
+        <li>Html</li>
+    </ul>
+</div>
+<div class="button-cont">
+    <input type="button" value="see project" class="potfolio-but">
+</div>
+</div>
+</div>        
+<section class="about-me">
```

Comment:
- [ ] You did well to wrap this section (about myself) inside the `<section>` tag 👏